packages/plugin-packages/mashroom-portal-remote-app-registry-k8s/src/js/jobs/ScanK8SPortalRemoteAppsBackgroundJob.ts

```diff
@@ -89,6 +89,16 @@ export default class ScanK8SPortalRemoteAppsBackgroundJob implements ScanBackgro
                                         status: 'Checking',
                                         lastCheck: Date.now(),
                                         error: null,
+                                        port,
+                                    };
+                                } else if (existingService.port !== port) {
```

Comment:
Couldn't we just add that condition to the first one? Because like this the condition body is duplicated